showcase/app/templates/components/form/index.hbs

```diff
@@ -0,0 +1,1078 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "Form Component"}}
+
+<Shw::Text::H1>Form / Layout components</Shw::Text::H1>
+
+<section data-test-percy>
+  <Shw::Text::H2>Containers</Shw::Text::H2>
+
+  <button type="button" class="shw-component-form-layout-button-highlight" {{on "click" this.toggleHighlight}}>
+    {{if this.showHighlight "Hide layout highlight" "Show layout highlight"}}
+  </button>
+
+  {{! Mix of shortcut names used for testing purposes }}
+  <div class="{{if this.showHighlight 'shw-component-form-layout-highlight-elements'}}">
+    <Shw::Grid @columns={{1}} @gap="2rem" as |SG|>
```

Comment:
[suggestion] general consideration: there's no need to have `Shw::Grid` containers with just one child (probably in other places too)